concent-vm/helper-scripts/concent-reset.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash -e
+
+# Confirm executing script
+read -p "Are you sure about execute this script? It's destroy databases, queues and other components like: Geth etc. [y/n]" -n 1 -r
+echo
+if [[ $REPLY =~ ^[Yy]$ ]]; then
+    databases_name=(
```

Comment:
Please print something if the answer is not `y`. E.g. `Operation canceled`. If the user did not type `n` but something else (he could have just pressed ENTER for example) he may be unsure whether the script did something or not.

Reply:
Done.